docs/README.md

```diff
@@ -1,153 +1,36 @@
 # Documentation
-### The workflow
-The Praqma workflow - often called The Phlow - is a combination of the Git extension and CI server for automation. In this the CI server is concourse with the git-phlow resource installed.
 
-![image](/docs/images/phlowmodel.svg)
+Table of contents
+- [Installation](/docs/installation.md)
+- [Workflow](/docs/workflow.md)
+- [Documentation](/docs/documentation.md)
+- [Configuration](/docs/configuration.md)
+- [Contribution guide](/CONTRIBUTING.md)
+- [Trouble shooting](/docs/troubleshooting.md)
 
-The top diagram Shows the steps the entire workflow entails.
-1. You do `git phlow workon` on an issue which has been groomed and is ready to become a feature in your project.
-2. You do `git phlow wrapup` and `git phlow deliver`, to deliver your changes to the remote repository. The branches will be prefixed with `delivery-branch-prefix` from the .gitconfig file.
-3. In the concourse Pipeline we are using the git-phlow tollgate resource, that automate the integration if it passes our toll-gate. It integrate branches prefixed with ready, meaning the job will trigger on such branches and merge the changes with the head of the integration.
-4. Now the tollgate tests will run in the job and if they pass, the already merged changes will be pushed to the integration branch. The ready branch will be deleted if everything goes well so it leaves things nice and clean. 
 
+#### Getting started
+To get started using this git extension we recommend to read the blogpost about git phlow and the entire automated workflow [here](https://www.praqma.com/stories/git-phlow/).
```

Comment:
WE should never have a "here" that's clickable we should put a meaningful description ie.
Read the [git-phlow blogpost(Linky McLinkyface).